pkg/plugins/common/kustomize/v1/init.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
Correct license years for all new files
```suggestion
Copyright 2021 The Kubernetes Authors.
```